src/AddtodoForm.js

```diff
@@ -1,20 +1,30 @@
 import React from "react";
 
+function AddTodoForm({onAddTodo}) {
 
-function AddTodoForm(props) {
+    let [todoTitle, setTodoTitle] = React.useState('');
+
+    function handleTitleChange(event) {
+        let newTodoTitle = event.target.value;
+        setTodoTitle(newTodoTitle);
+    }
 
     function handleAddTodo (event){
         event.preventDefault();
-        const todoTitle = event.target.elements.title.value;
-        props.onAddTodo(todoTitle);
+        let newTodo = {
+            title: todoTitle, 
+            id: Date.now() 
+          };
+        onAddTodo(newTodo);
```

Comment:
To help with readability you could compact lines 14-18 into one line.

```suggestion
        onAddTodo({title: todoTitle, id: Date.now()});
```

Reply:
Thank you for all of your suggestions! I appreciate them. 😄

Reply:
Anytime. Don't hesitate to reach out if you have any questions.